src/test/java/com/aliyuncs/fc/FunctionComputeClientTest.java

```diff
@@ -111,6 +107,7 @@
     private static final String TRIGGER_NAME = "testTrigger";
     private static final String TRIGGER_TYPE_OSS = "oss";
     private static final String TRIGGER_TYPE_LOG = "log";
+    private static final String TRIGGER_TYPE_TIMER = "timer";
```

Comment:
Seems we call the trigger "time" trigger everywhere, why we make the type "timer" here?

Reply:
the type of the trigger is called "timer" but the service is called "time trigger"